### PR TITLE
fix(frontend): checkbox partially clickable

### LINKED ISSUE #3792: [Préavis de retraite] Les boutons radios accompagnés du textes sont difficiles à cliquer
# Bug

Screenshot 2021-09-29 at 16 24 14

# Proposition de correction

Il faut que la zone ensemble soit clickable

### PR DESCRIPTION
**Pull request by maxgfr**
fix #3792

### CODE REVIEW

File: packages/code-du-travail-frontend/src/outils/common/Wizard.js [hunk 1]
```diff
@@ -145,7 +145,12 @@ function Wizard({
         {({ handleSubmit, form, invalid, submitFailed }) => {
           return (
             <>
-              <StyledForm onSubmit={handleSubmit}>
+              <StyledForm
+                onSubmit={() => {
+                  event.preventDefault();
```
Comment (carolineBda) [inline packages/code-du-travail-frontend/src/outils/common/Wizard.js]: on a besoin de ça ?

Comment (maxgfr) [reply packages/code-du-travail-frontend/src/outils/common/Wizard.js]: Hmmm 🤔

Comment (maxgfr) [reply packages/code-du-travail-frontend/src/outils/common/Wizard.js]: Non

**Review by carolineBda — APPROVED** [review]
👍

**Pull request merged**